data/human/culture conversations.txt

```diff
@@ -1101,3 +1101,279 @@ mission "Little Vienna in New Austria"
+		"The venue quiets as spectators scatter, leaving you to drift back to your <model> in the calm that's followed the thrill of the race."
+
+mission "A Sunracer Hovercraft Racing"
+	minor
+	repeat
+	invisible
+	to offer
+		random < 40
+	source "Sunracer"
+	substitutions
+		"<racename>" "${adjectives} ${racetrack}"
+		"<megaparsec name>" "${civilian}"
+		"<syndicate name>" "${syndicate fighter}"
+		"<delta name>" "${free worlds fighter}"
+		"<deep name>" "${deep fighter}"
+	on offer
+		conversation
+			`${sunracer opening}, a ${sunracer hovercraft adhook}: there's a hovercraft race at the <racename>, just a few hours away by hover rail, or much faster aboard a <model>. ${sunracer hovercraft adcloser}.`
+			choice
+				`	(Check it out.)`
+				`	(Skip the hovercraft race.)`
+					decline
+			action
+				
+				"race length" = 3 + "roll: 2"
+				"syndicate boost" = 2 + "roll: 2"
+				"delta boost" = 4 + "roll: 8"
+				"deep boost" = 6 + "roll: 12"
+				# The ( ( 10 - x + "race length" ) / 10 ) flips to 1 when race length is x+, 0 if less than x. Used to only roll for the length of the track.
+				"megaparsec speed" = "roll: 12" + "roll: 12" + "roll: 12" + ( ( 10 - 4 + "race length" ) / 10 ) * ( "roll: 12" ) + ( ( 10 - 5 + "race length" ) / 10 ) * "roll: 12"
+				"syndicate speed" = "roll: 10" + "roll: 10" + "roll: 10" + ( ( 10 - 4 + "race length" ) / 10 ) * ( "roll: 10" ) + ( ( 10 - 5 + "race length" ) / 10 ) * "roll: 10"
+				"delta speed" = "roll: 8" + "roll: 8" + "roll: 8" + ( ( 10 - 4 + "race length" ) / 10 ) * ( "roll: 8" ) + ( ( 10 - 5 + "race length" ) / 10 ) * "roll: 8"
+				"deep speed" = "roll: 6" + "roll: 6" + "roll: 6" + ( ( 10 - 4 + "race length" ) / 10 ) * ( "roll: 6" ) + ( ( 10 - 5 + "race length" ) / 10 ) * "roll: 6"
+				"syndicate laps" = "syndicate speed" + "syndicate boost"
+				"delta laps" = "delta speed" + "delta boost"
+				"deep laps" = "deep speed" + "deep boost"
+			`	You board <ship> for the quick trip to the <racename>. Out the window, ${sunracer city to wilderness description}. Eventually, you park your ship outside the <racename>, and walk up to the racetrack's entrance.`
+			label decisioning
+			`	The screen above flashes race details:`
+			`		<racename>: &[race length] Hour Race`
+			``
+			`		Today's Racers:`
+			`		Megaparsec Racing Technologies: <megaparsec name>`
+			`		Syndicate Performance Racing: <syndicate name>`
+			`		Delta V Grand Prix: <delta name>`
+			`		Deep Sky Dynamics: <deep name>`
+			``
+			choice
+				`	(Watch the race.)`
+					goto race
+				`	(Ask someone for an overview of hover racing.)`
+				`	(Use your ship to scan the racers.)`
+					to display
+						or
+							"flagship attribute: outfit scan power" > 0
+							"flagship attribute: tactical scan power" > 0
+					goto scanner
+			`	Near the <racename>'s entrance, you spot a friendly figure ${sunracer fan tshirt}. Though clearly not a racer, he clearly is very familiar with hovercraft racing. You ask him for an overview and he responds:`
```

Comment:
Hmm... it might be interesting if the fan's gender switched between male/female/unknown each time. That might require a bit more work, but I think it would make the mission slightly more interesting and make the fan feel less like a stereotypical sports nerd.

Reply:
That is a nice idea - let's see if I can figure out how to make it work with a multi-line `replace`. If I fail, I don't think its a _huge_ deal since the player presumably won't ask for an overview every time they visit the track, but I agree that would be better.